### DISCUSSION TITLE
Can you add multi device select menu from blobsaver?

### DISCUSSION THREAD

**Discussion by garyakimoto** [airsquared/blobsaver#636] [Ideas]
I think there are so many peoeple like me have over 1 iPhone...
so could you add a menu inside blobsaver and let us "Read from device" to read ECID?

I know there are some tech problem if you add "Read Apnoce" from multi devic(because it may reboot wrong device), so this is not the one I request, I need need read ecid function

Thanks

**Comment by airsquared** [comment]
You want an option to read from multiple devices at once?

**Reply by garyakimoto** [reply]
only the drop download menu to select one of iPhone to read ECID without unplug them.

Thanks
